### DISCUSSION TITLE
Read only field visible in edit view

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16540] [How to]
I would like to show read-only field (for certain profile) in record edit view. Show but not allow to edit. The thing is here is some instructions related to record in text field, but it will be handy if user can read 'em while editing record (not amend). 
Is it possible?

Best

**Comment by mariuszkrzaczkowski** [comment]
from the panel you will set the field to be always read-only in editing for everyone, if it is to be only for specific profiles, then two ways:
1. expansion of profiles, the best way, but requires more time
2. handler or engine modification, quick change
